# This message hold the state of the emergency stop button

std_msgs/Header header
bool emergency_stop_activated                         # True if the emergency stop has been activated, false otherwise
